def add():
    a = 4
    b = 5
    print(a + b)
    raise Exception
add()